prompt_translation/03_create_dashboard.ipynb

```diff
@@ -28,12 +28,15 @@
     "- `SOURCE_WORKSPACE`: The workspace id of the source dataset (unless otherwise specified, this will be `admin`)\n",
```

Comment:
<p>Make sure to update the  looks like a unfinished sentence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/data-is-better-together/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>